src/helper-modules/govuk-notify/govuk-notify.service.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  BadRequestException,
+  ForbiddenException,
+  Injectable,
+  InternalServerErrorException,
+  UnauthorizedException,
+  UnprocessableEntityException,
+} from '@nestjs/common';
+import { PinoLogger } from 'nestjs-pino';
+import { NotifyClient } from 'notifications-node-client';
+
+import { PostEmailsRequestItemDto } from '../../modules/emails/dto/post-emails-request.dto';
+import { PostEmailsResponseDto } from './dto/post-emails-response.dto';
+
+@Injectable()
+export class GovukNotifyService {
+  constructor(private readonly logger: PinoLogger) {}
+
+  /**
+   * Send email to one recipient using GOV.UK template id
+   * @param {String} govUkNotifyKey
+   * @param {PostEmailsRequestItemDto} post email request item
```

Comment:
```suggestion
   * @param {PostEmailsRequestItemDto} email post email request item
```

Reply:
Just double checking, is suggestion to have `email post email request item`?

Reply:
Sorry, I realise that this suggestion was unclear. The reason I suggested that was the JSDoc seems to interrept the first word after the type as the name of the parameter, which at the moment is `email`. So, at the moment, I'm getting `JSDoc '@param' tag has name 'post', but there is no parameter with that name.` as a warning. However, I think it would actually be better to change the parameter name to `postEmailsRequestItem`, so this line could become:
`@param {PostEmailsRequestItemDto} postEmailsRequestItem`

Reply:
Thanks, I forgot to check how JSDoc works.

I ended up using `@param {PostEmailsRequestDto} postEmailsRequest` . Request used to be array and have `items`, but I changed to be single object, so no need for `item`. Hope it works.